### PR TITLE
Update RSA key search to find larger keys

### PR DESCRIPTION
**Pull request by sylvainpelissier**
Updated code to find larger keys within the 256 bytes buffer given as parameter. Tested with [#2020](https://github.com/radareorg/radare2-regressions/pull/2020).

### CODE REVIEW

File: libr/search/rsa-find.c [hunk 1]
```diff
@@ -14,68 +14,90 @@ Parses a single field of the key, beginning at start.  Each field
...
+	for (i = 2; i < len - sizeof (versionmarker); i++) {
 		if (memcmp (&buf[i], versionmarker, sizeof (versionmarker))) {
 			continue;
 		}
 
-		index = 0;
-		for (k=i; k != 0 && k > i - 20; k--) {
+		index = -1;
+        // Going backward maximum up to 5 characters.
+        if (i < 5) {
```
Comment (radare) [inline libr/search/rsa-find.c]: Wrong indent.

File: libr/search/rsa-find.c [hunk 2]
```diff
@@ -14,68 +14,90 @@ Parses a single field of the key, beginning at start.  Each field
...
+            max = i;
+        }
```
Comment (radare) [inline libr/search/rsa-find.c]: The else goes here

File: libr/search/rsa-find.c [hunk 3]
```diff
@@ -14,68 +14,90 @@ Parses a single field of the key, beginning at start.  Each field
...
+        else {
+            max = 5;
+        }
+		for (k=i-2; k >= i - max; k--) {
```
Comment (radare) [inline libr/search/rsa-find.c]: Spaces around =

**Comment by Maijin** [comment]
@sylvainpelissier try the ./sys/indent.sh script

**Comment by sylvainpelissier** [comment]
> sys/indent.sh

Indeed, the script works fine.

**Pull request merged**